src/opengl_tex.c

```diff
@@ -368,9 +368,11 @@ static GLuint gl_loadSurface( SDL_Surface* surface, unsigned int flags, int free
    }
    else {
       *vmax = 1.;
-      glPixelStorei( GL_UNPACK_ALIGNMENT, MIN( surface->pitch&-surface->pitch, 8 ) );
+      SDL_Surface *s = SDL_ConvertSurfaceFormat( surface, SDL_PIXELFORMAT_ARGB8888, 0 );
```

Comment:
It seems like I get the colours inverted here, but it works with `SDL_PIXELFORMAT_ABGR8888`... You don't have an issue with this?